src/components/journeys/editor-journey/editor-journey.jsx

```diff
@@ -198,6 +198,35 @@ const EditorJourney = ({onActivateDeck, setCanViewTutorialsHighlight, setShowJou
         () => ({
             popoverClass: 'gui-journey',
             overlayOpacity: 0,
+            onDestroyStarted: () => {
+                let stepName = '';
+                if (driverObj.getActiveIndex() === 0) {
+                    stepName = 'pick-genre-step';
+                } else if (driverObj.getActiveIndex() === 1) {
+                    stepName = 'game-step';
+                } else if (driverObj.getActiveIndex() === 2) {
+                    stepName = 'animation-step';
+                } else if (driverObj.getActiveIndex() === 3) {
+                    stepName = 'music-step';
+                } else if (driverObj.getActiveIndex() === 4) {
+                    stepName = 'clicker-game-step';
+                } else if (driverObj.getActiveIndex() === 5) {
+                    stepName = 'pong-game-step';
+                } else if (driverObj.getActiveIndex() === 6) {
+                    stepName = 'animate-character-step';
+                } else if (driverObj.getActiveIndex() === 7) {
+                    stepName = 'make-fly-animation-step';
+                } else if (driverObj.getActiveIndex() === 8) {
+                    stepName = 'record-sound-step';
+                } else if (driverObj.getActiveIndex() === 9) {
+                    stepName = 'make-music-step';
+                }
```

Comment:
Do we have a specific reason to avoid using an array/object that contains the index-to-stepName mapping? We can have the same behavior here by implementing something like
```js
// root level
const STEP_NAMES = ['pick-genre-step', 'game-step', ..., 'make-music-step'];
...
// in place of the current code
onDestroyStarted: () => {
    const stepName = STEP_NAMES[driverObj.getActiveIndex()] || '';
...
```